app/src/main/java/cm/aptoide/pt/app/view/AppViewFragment.java

```diff
@@ -1041,45 +1017,6 @@ private void destroyAdapter(MoPubRecyclerAdapter adapter) {
     }
   }
 
-  @Override public void recommendsShare(String packageName, Long storeId) {
-
-    AptoideApplication application = (AptoideApplication) getContext().getApplicationContext();
-    TimelineAnalytics analytics = application.getTimelineAnalytics();
-    if (application.isCreateStoreUserPrivacyEnabled()) {
-      SocialRepository socialRepository = RepositoryFactory.getSocialRepository(getActivity(),
-          application.getDefaultSharedPreferences());
-      LayoutInflater inflater = LayoutInflater.from(getActivity());
-      AlertDialog alertDialog = new AlertDialog.Builder(getActivity()).create();
-      View alertDialogView = inflater.inflate(R.layout.logged_in_share, null);
```

Comment:
If you search for logged_in_share in project you will find more stuff to be deleted (variables & methods)